sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/NotificationHookLiveTests.cs

```diff
@@ -453,10 +453,6 @@ await foreach (NotificationHook hook in adminClient.GetHooksAsync())
                     var webHook = hook as WebNotificationHook;
 
                     Assert.That(webHook, Is.Not.Null);
-                    Assert.That(webHook.CertificateKey, Is.Not.Null);
-                    Assert.That(webHook.CertificatePassword, Is.Not.Null);
-                    Assert.That(webHook.Username, Is.Not.Null);
-                    Assert.That(webHook.Password, Is.Not.Null);
```

Comment:
The service behaves incredibly weirdly here. Usually they default strings not specified by the user as empty strings. In a webhook, this is a bit different. If not specified, the properties will be `undefined`, so they won't be returned. If you specify them at least once, this changes forever. If you set one of them to null to reset to their default state, they will be reset to empty strings.

Since we can't guarantee they won't be null here (we're getting a list of random hooks from the resource), removed the checks.

Reply:
sounds good for the test. I don't think it is worth adding this to the docstrings. do u?

Reply:
It seems more like accidental behavior than something they actually wanted to do. But I don't it's really worth it. Users will be setting properties to specific values (`Property = new_value`), or resetting them to the default value (`Property = null`), but they won't really be default-checking them.